pyof/foundation/basic_types.py

```diff
@@ -412,7 +412,7 @@ class BinaryData(GenericType):
     return the size of the instance using Python's :func:`len`.
     """
 
-    def __init__(self, value=b''):  # noqa
+    def __init__(self, value=None):  # noqa
```

Comment:
this is already covered here https://github.com/kytos/python-openflow/blob/master/tests/test_foundation/test_basic_types.py#L169 , isn't it?
I don't think adding ``None`` instead of leaving the default initialization is of much use... All other ``GenericTypes`` accept ``None``, and we are testing none of them.
If you think it is important to add tests for ``GenericType`` accepting ``None`` explicitly, maybe we should write one that does the test for all of them.